src/internal/Subject.ts

```diff
@@ -8,6 +8,12 @@ import { SubjectSubscription } from './SubjectSubscription';
 import { rxSubscriber as rxSubscriberSymbol } from '../internal/symbol/rxSubscriber';
 
 /**
+ * A Subject is a special type of Observable that allows values to be
```

Comment:
I think here is a minor error. This part belongs to the Subject class, directly below the SubjectSubscriber